README.md

```diff
@@ -309,6 +309,10 @@ The following options are supported by all filters except `Callback` and `Finder
 - `mode` (`string`, defaults to `OR`) The conditional mode to use when searching for
   multiple values. Valid values are `OR` and `AND`.
 
+#### `Finder`
+
+- `finder` (`string`) The name of the finder method to use, without the 'find' prefix.
```

Comment:
Telling someone how to call a method which your code executes, is not duplicating the documentation. It's documenting how your software works. Especially if there is possible ambiguity.